pkg/pillar/cmd/zedmanager/handleverifier.go

```diff
@@ -252,27 +260,75 @@ func handleVerifyImageStatusDelete(ctxArg interface{}, key string,
 	log.Infof("handleVerifyImageStatusDelete done for %s\n", key)
 }
 
-// check whether the cert files are installed
-func checkCertsForObject(safename string, ss *types.StorageStatus) bool {
-
-	var cidx int = 0
-
+// whether the object contains certs reference
+func containsCerts(safename string, ss *types.StorageStatus) bool {
```

Comment:
Does it make sense to be a member of types.StorageStatus ? 
   ss.getCertificateCount()

Then you cna just check ss.getCertificateCount() == 0

Reply:
Done.